<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersDashboardClass</class>
 <widget class="QWidget" name="UsersDashboardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UsersDashboard</string>
  </property>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>91</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_11{
color:black;
  border-radius: 25px;


font-size:15px;
}</string>
   </property>
   <property name="text">
    <string>All Users :</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>791</width>
     <height>561</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>789</width>
      <height>559</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>761</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
  border-radius: 15px;
  border: 2px solid white ;
background-color:white;

}
QLabel
{
font-size:15px;
font-weight:bold;
color:#2B2A2A;
}
QPushButton{
  border-radius: 10px;
  background: #FF7F7F;
color:white;



}

QPushButton:hover{
background: #CCD0CD;
color:Black;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>3</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>#12345678901234</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>3</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ahmed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>3</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ghazaly</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>3</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>012345678900</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>5</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="edit-delete"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_43">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>5</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Assets/profile.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
  border-radius: 15px;
  border: 2px solid white ;
background-color:white;

}
QLabel
{
font-size:15px;
font-weight:bold;
color:#2B2A2A;
}
QPushButton{
  border-radius: 10px;
  background: #FF7F7F;
color:white;



}

QPushButton:hover{
background: #CCD0CD;
color:Black;
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>3</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>#ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>3</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>3</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>3</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>731</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
font-size:34px;
font-weight:bold;
color:#2B2A2A;
}</string>
   </property>
   <property name="text">
    <string>All Users</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
